src/test/java/bb/example/PersonDatabaseTest.java

```diff
@@ -1,42 +1,52 @@
 package bb.example;
 
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.mockito.InjectMocks;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
+@ExtendWith(MockitoExtension.class)
 class PersonDatabaseTest {
+
     @Mock
-    PersonI mockedPerson;
-    @InjectMocks
+    PersonI person;
+
     PersonDatabase database;
 
     @BeforeEach
-    void setUp() {
-        MockitoAnnotations.openMocks(this);
+    void beforeEach() {
+        database = new PersonDatabase();
     }
 
     @Test
-    void testAddPerson() {
-        when(mockedPerson.getId()).thenReturn(1);
-        database.addPerson("John", "Doe", "123456789", 25);
-
-        verify(mockedPerson).setName("John");
-        verify(mockedPerson).setSurname("Doe");
-        verify(mockedPerson).setPhone("123456789");
-        verify(mockedPerson).setAge(25);
-
-        PersonI addedPerson = database.findById(1);
-
-        assertThat(addedPerson).isNotNull();
-        assertThat(addedPerson.getName()).isEqualTo("John");
-        assertThat(addedPerson.getSurname()).isEqualTo("Doe");
-        assertThat(addedPerson.getPhone()).isEqualTo("123456789");
-        assertThat(addedPerson.getAge()).isEqualTo(25);
+    @DisplayName("Should add person to database")
+    void shouldAddPersonTest() {
+        when(person.getId()).thenReturn(1);
+        when(person.getName()).thenReturn("John");
+        when(person.getSurname()).thenReturn("Doe");
+        when(person.getPhone()).thenReturn("123456789");
+        when(person.getAge()).thenReturn(25);
+
+        PersonI foundPerson = database.findById(1);
+        assertThat(foundPerson).isNull();
+
+        database.addPerson(person);
+
+        foundPerson = database.findById(1);
+
+        assertThat(foundPerson).isNotNull();
+        assertThat(foundPerson.getId()).isEqualTo(1);
+        assertThat(foundPerson.getName()).isEqualTo("John");
+        assertThat(foundPerson.getSurname()).isEqualTo("Doe");
+        assertThat(foundPerson.getPhone()).isEqualTo("123456789");
+        assertThat(foundPerson.getAge()).isEqualTo(25);
+
+        verify(person).setId(1);
+        verifyNoMoreInteractions(person);
```

Comment:
If there were more operations called on the mock, the test will fail in this line